### DISCUSSION TITLE
What does “entry” refer to?

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [WeblateOrg/weblate#7137] [Q&A] [answered]
What I want to know is the meaning of each “entry” in following strings. They are all not related to glossary or translation memory.
Strings, units or something else?
This clarification would help my translation work.

- [Entry by the same name already exists.](https://hosted.weblate.org/translate/weblate/application/en/?checksum=c22fc884c79c7840)
- [Showing and replacing only the first 250 matches. Perform the replacement again to replace more entries.](https://hosted.weblate.org/translate/weblate/application/en/?checksum=69895b99250f78a5)
- [Marking all :ref:`appstore` changelog entries read-only](https://hosted.weblate.org/translate/weblate/documentation/en/?checksum=c2fde739bea2bd6)
- [The uploaded file is merged to update the translation, overwriting existing entries by default (this can be turned off or on in the upload dialog).](https://hosted.weblate.org/translate/weblate/documentation/en/?checksum=23de0e467ad85e6d)
- [Checks that semicolons at the end of sentences are replicated between both source and translation. This can be useful to keep formatting of entries such as desktop files.](https://hosted.weblate.org/translate/weblate/documentation/en/?checksum=6514e14e92a7625c)

**Comment by Geeyun-JY3** [comment]
BTW, I think that document should explicitly tell users(especially translators) what “entry” in glossary and TM is.

**Comment by nijel** [comment]
At least some of these deserve rewriting, I've fixed some in e5a60c0d87

**Comment by Geeyun-JY3** [comment]
What does the “desktop file” in [Checks that semicolons at the end of sentences are replicated between both source and translation. This can be useful to keep formatting of entries such as desktop files.](https://hosted.weblate.org/translate/weblate/documentation/en/?checksum=6514e14e92a7625c) refer to?

**Reply by nijel** [reply]
`.desktop` files, see https://wiki.archlinux.org/title/desktop_entries. But is probably misleading here as it is definitely not the common use-case. Removed in 8616237ae0

**Comment by Geeyun-JY3** [comment, marked as answer]
According to the context in [document](https://docs.weblate.org/en/latest/admin/addons.html#id2), I think the `entries` (entry) in [Marking all :ref:`appstore` changelog entries read-only](https://hosted.weblate.org/translate/weblate/documentation/en/?checksum=c2fde739bea2bd6) can be changed to `strings` (string).

**Reply by nijel** [reply]
Changed in 5e79483253
